formula_sae_esf.tex

```diff
@@ -1538,8 +1538,7 @@ \section{Additional LV-parts Interfering with the Tractive System}
 
         \subsubsection{Description}
 
-            The GLV system will be powered via a DC/DC converter which will convert the 100V system to a 13.5V line for powering CAN nodes and other LV devices. This voltage will be regulated down to 12V and 5V for the LV devices. Before the DC-DC converter, there is a fuse for the GLV system as a whole, with a rating of 4A.
-            %And will the 5V line be fused?
+            \hlr{The GLV system will be powered via a DC/DC converter which will convert the 100V system to a 13.5V low voltage system. Before the DC-DC converter, there is a fuse for the GLV system as a whole, with a rating of 4A.}
```

Comment:
There is a 1A fuse (the same one for the TSMP wires) on the AIR control board, that is protecting the TS lines that power the DC/DC converter. On the GLV side of the DC/DC converter there is a 4A fuse protecting the wires that run to each of the shutdown buttons and GLVMS.